src/absil/il.fs

```diff
@@ -2193,6 +2198,8 @@ and ILExportedTypesAndForwarders =
 
     member x.AsList = let (ILExportedTypesAndForwarders ltab) = x in Map.foldBack (fun _x y r -> y :: r) (ltab.Force()) []
 
+    member x.TryFindByName nm = let (ILExportedTypesAndForwarders ltab) = x in (ltab.Force()).TryFind nm
```

Comment:
Best not to use one-line formatting.  I know the line above does but it's ancient grotty code and shouldn't :)

Reply:
Makes sense :)